lang/localize-core-element.js

```diff
@@ -9,6 +9,9 @@ export const LocalizeCoreElement = superclass => class extends LocalizeMixin(sup
 				case 'ar':
 					translations = await import('./ar.js');
 					break;
+				case 'cy':
```

Comment:
If the language set is `cy-gb`, will the `langs` object passed in have both `cy` and `cy-gb`?

Reply:
yup it would get passed [`cy-gb`, `cy`, `en`] (and I think `en-us` before `en` but I don't recall for sure)